src/main/java/com/gdsc/studyjamapi/entity/User.java

```diff
@@ -36,7 +36,7 @@ public class User extends BaseEntity implements UserDetails {
   @Column(nullable = false)
   @Enumerated(EnumType.STRING)
   private Role role;
-
+  @Builder.Default
```

Comment:
Why did you put this annotation here?

Reply:
Because I think when the account first created, it must be default enabled for users